### ISSUE TITLE
Fix critical bug: forms with recaptcha can not be sended and onoffice-captchacontrol.js is also loaded in forms without recaptcha

### ISSUE THREAD

**Issue by andernath** [onOffice-Web-Org/oo-wp-plugin#836]
Changes in #774 causes errors with recaptcha in forms.

1. Critical: If I open a form with recaptcha true, an**Uncaught TypeError: onOffice.captchaControl is not a function** occurs. It is not possible to send this form.
Example: https://w3pm.onofficeweb.com/formulare/eigentuemer/
2. If I open a form without recaptcha, the onoffice-captchacontrol.js will be loaded anyway
Example: https://w3pm.onofficeweb.com/formulare/kontaktformular/

I think one problem is ScriptloaderGenericConfigurationDefault.checkCaptchaActive(array $forms) always returns true in line 236.

I can not release v4.20 with this bug. Could you please check and fix it @dai-eastgate ?

**Comment by dai-eastgate** [comment]
@andernath I will check it

**Comment by yeneastgate** [comment]
> 1. Critical: If I open a form with recaptcha true, an**Uncaught TypeError: onOffice.captchaControl is not a function** occurs. It is not possible to send this form.

I have investigated the cause of this error. This error occurs because onOffice.captchaControl is loading in the footer. But in the Form Submit feature, onOffice.captchaControl is being called so this JS file must be loaded in the header.


> I think one problem is ScriptloaderGenericConfigurationDefault.checkCaptchaActive(array $forms) always returns true in line 236.

You're right, I checked and it doesn't work when deactivating "Requires Captcha"

@andernath I have fixed it and this is demo video:

https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/104892108/d7984794-2cd0-449c-ae75-007aa4f66f9c
